contracts/scripts/testing.ts

```diff
@@ -0,0 +1,217 @@
+import rollupJson from "./util/Rollup.json";
+import sequencerInboxJson from "./util/SequencerInbox.json";
+import { Wallet, utils, ethers, BigNumber } from "ethers";
+import assert from "assert";
+import fs from "fs";
+import path from "path";
+
+const ROOT_DIR = __dirname + "/../../";
+const CONFIGURE_SYSTEM_PATH = path.resolve(
+  __dirname,
+  "../../clients/geth/specular/sbin/configure_system.sh"
+);
+
+function delay(seconds: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, seconds * 1000));
+}
+
+function getSequencerAddress(data: string): string {
+  const sequencerAddrMatch = data.match(/export SEQUENCER_INBOX_ADDR=(.*)/);
```

Comment:
Can you get it from the environment variable? If not just hard-encode here.

Reply:
It's hardcoded